pagopa/io-app
# fix(IT Wallet): [SIW-1778] Handle 401 status code during Wallet Instance revocation within eID issuance

## Short description
This PR addresses the lack of 401 status code handling during the Wallet Instance revocation state in the eID issuance machine.

## List of changes proposed in this pull request
- Removed redundant `SessionExpired` state. Machines now directly invoke the `handleSessionExpired` action to manage session expiration, transitioning to the required state without an intermediary state.
- Added session expired handling in `WalletInstanceRevocation` state
- Added tests

## How to test
- Using a proxy (e.g.: Proxyman) intercept the `POST /wallet-instances` and `PUT /wallet-instances/current/status` endpoint response and return a `401` status code
- Start the IT Wallet activation flow
- Check that the refresh session screen is correctly shown after the TOS acceptance
- Check that after the session refresh screen it is possibile to continue the activation flow
- Check that the session refresh screen is shown when trying to revoke the wallet instance
